browser/src/VariantPage/VariantPage.tsx

```diff
@@ -148,7 +148,7 @@ export type Population = {
   ac_hemi: number | null
   ac_hom: number
   homozygote_count?: number
-  hemizygote_count?: number | null
+  hemizygote_count?: number
```

Comment:
Ah what I meant here was let's use the `| null` rather than `?` in both counts--that's assuming that we're getting those values directly from GraphQL, if not then never mind.

Reply:
Aha, I see, yeah my edit was unclear also. The query for the Variant page doesn't actually request the `hemizygote_count` and `homozygote_count` fields, so I added the `?` to let me Type the populations here without having to add a dummy value for those two. So maybe the true type should actually be `hemizygote_count?: number | null`?

```
populations {
  id
  ac
  an
  ac_hemi
  ac_hom
}
``` 

Digging a little deeper, looks like `ac_hemi` and `ac_hom` are aliased names of `hemizygote_count` and `homozygote_count` that you can query to get the same value. Looks like ClinVar names them `ac_hemi` etc, so it made sense to add the alias in our API at some point in the past to match.

Unsure of what the best thing to do right now is. It feels like it should be safe to remove `hemizygote_count` and `homozygote_count` from this type (and a brief run of tests and typecheck after doing that goes fine), but I also feel like maybe that should be a seperate PR? It feels different in scope than this one, but I can also see the argument that this PR concerns this type and these files.

Reply:
Ok nice, that makes sense to me.

Added that commit and going to formally re-request review! I did a force push so the changes since last reviewed might not work, but all the PR review commits are tagged as fixup.